srtcore/srt_c_api.cpp

```diff
@@ -314,4 +314,16 @@ int srt_listen_callback(SRTSOCKET lsn, srt_listen_callback_fn* hook, void* opaq)
     return CUDT::installAcceptHook(lsn, hook, opaq);
 }
 
+int srt_getversion(void* val, int len)
```

Comment:
Why a void * for val if the only len value supported is the one of uint32_t? For my education. 
API could have been like the time(2) function (uint32_t srt_getversion(uint32_t* val); with optional val (can be NULL). Simpler to use for lazy programmer but may be inconsistent with the rest of the API.

Reply:
This might be an overcomplication. I assumed that we might want to add the Git Commit ID in future, if `len` is set to `sizeof(uint32_t) + sizeof(uint64_t)`.

Reply:
Then we'll add a new function for that purpose. This will be for something different and more complicated, as well as the API that requires a variable to fill is more clumsy and user-hostile.